### ISSUE TITLE
[Assistive tech and device support] Content causes the user's keyboard focus to get trapped (11.06.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#64658]
#### VFS team name
Veteran Facing Forms

#### VFS product name
Product 21-0972 Alternate Signer

#### VFS feature name
product 21-0972 alternate signer

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Content causes the user's keyboard focus to get trapped

### Issue Details
On the memorable date component (eg. [/alternate-signer-form-21-0972/veteran-personal-information](https://staging.va.gov/supporting-forms-for-claims/alternate-signer-form-21-0972/veteran-personal-information)), when the component is in an error state keyboard focus is trapped inside the component.

### VA.gov Experience Standard
[Category Number 11, Issue Number 06](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.1.2_A

### Platform Recommendation
Users should be able to navigate away from the input even if they've provided an invalid response. Note that this was reported to the Design System Team in #64284.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by jeana-adhoc** [comment]
Keeping this open until I can validate the fix in staging

**Comment by powellkerry** [comment]
@jeana-adhoc This has been deployed to staging. Please verify.

**Comment by jeana-adhoc** [comment]
@powellkerry - This looks good, the keyboard trap has been cleared. 

However, I'm now noticing an error message issue, that I'll file a new issue for so this one can be closed. 

Thanks!

**Comment by jeana-adhoc** [comment]
New issue for [memorable date month select error message](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2121)
